### PR TITLE
Add header #5

### PR DESCRIPTION
**Pull request by Ebano**

### CODE REVIEW

File: app/views/tasks/index.html.erb [hunk 1]
```diff
@@ -33,3 +41,36 @@
 <br>
 
 <%= link_to 'New Task', new_task_path %>
+
+
+<!-- <table>
```
Comment (eulenfrau) [inline app/views/tasks/index.html.erb]: ??!! WTF??!!

File: app/views/tasks/index.html.erb [hunk 2]
```diff
@@ -1,6 +1,14 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Tasks</h1>
+<title>To-Do-List</title>
+
+
+<header>
```
Comment (eulenfrau) [inline app/views/tasks/index.html.erb]: well done!

**Review by eulenfrau — CHANGES_REQUESTED** [review]
please remove unused code, honey!

**Review by bascht — APPROVED** [review]
Marvellous! :sparkles:

**Review by MarMarMet — APPROVED** [review]
Wunderbar!

**Pull request merged**